event_sidecar/src/event_stream_server/sse_server.rs

```diff
@@ -977,23 +980,6 @@ mod tests {
         }
     }
 
-    #[tokio::test]
-    async fn should_filter_duplicate_main_events() {
```

Comment:
Some of these tests fail because the legacy endpoints have a different output than input (due to translation). I will bring these tests back in the next PR